scripts/rspec/create-github-issue.ps1

```diff
@@ -23,6 +23,10 @@ param (
 Set-StrictMode -version 2.0
 $ErrorActionPreference = "Stop"
 
+if ($language -eq "vbnet") {
```

Comment:
Why making a check and throw instead of removing the script parameter?

Reply:
I thought about that, but that would make it easier to make a mistake (e.g. forgetting to specify vb as a language, as we do not work too often with it).

I don't want to touch the logic of the script, as we may add support for JIRA at some point, or move VB to github. It's a dev script, so went for simple.